doc/github-actions.md

```diff
@@ -0,0 +1,26 @@
+# GitHub Actions
+
+## Configuration
+
+his project supports github actions for continuous integration and delivery. To enable GitHub actions on this repo, you will
+have to create the following secrets:
+
+```bash
+SUPABASE_ACCESS_TOKEN: Your personal access token, can be found at https://supabase.com/dashboard/account/tokens
+SUPABASE_DB_PASSWORD: Your supabase database password
+SUPABASE_PROJECT_ID: Your supabase project id
+SUPABASE_URL: Your supabase project URL
+SUPABASE_ANON_KEY: Your supabase project anonymous key
+```
+
+Also, you will need to configure the some variables:
+```bash
+VITE_IS_DEMO: Set to `true` if you want to display the demo banner
+```
+
+## Deploying to Another Repository
+
+If you want to deploy the static website to another repository, you can configure the following secret on you repository:
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** Shouldn't we do a setup on the target repot? Can we just magically push on it?‏

Reply:
To be honest, I don't konw if this will work directly, I will need to test it to be sure